### DISCUSSION TITLE
Relative Link outside Remix

### DISCUSSION THREAD

**Discussion by jaschaio** [remix-run/remix#3910] [Q&A] [answered]
I have two remix applications, one of the runs in a subdirectory under `/docs` and the rest at root.

I have `nginx` in front of the application to take care of routing requests to the right place.

The problem is if I follow a relative link to `/docs` remix doesn't request the server but tries to render it itself, which leads to a 404 error.

Any way I can render a `<a href="">` component without remix taking over the routing?

**Comment by machour** [comment, marked as answer]
You can either use a simple `<a href>`, or use `<Link reloadDocument />`

**Reply by jaschaio** [reply]
I actually used a simple `<a href>` but it loaded so fast that it looked like it was processed by remix but it actually wasn't so you are correct. Still good to know about `<Link reloadDocument />`, thanks!
